### DISCUSSION TITLE
pulsar java client 3.2.1 version ,   use  pulsar client api 3.1.3 version?  no same version ??

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#22328] [Q&A]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### Read release policy

- [X] I understand that unsupported versions don't get bug fixes. I will attempt to reproduce the issue on a supported version of Pulsar client and Pulsar broker.

### Version

pulsar java client 3.2.1

### Minimal reproduce step

idea log :
![image](https://github.com/apache/pulsar/assets/49554320/7811d03b-76cc-4342-8bff-885aa4218088)

idea show :
![image](https://github.com/apache/pulsar/assets/49554320/1b894f32-b819-4948-a739-c19524a5774d)

### What did you expect to see?

?

### What did you see instead?

?

### Anything else?

?

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

**Comment by lhotari** [comment]
This doesn't seem to be a bug.

In Pulsar 3.2.x, you can use [Pulsar BOM](https://github.com/apache/pulsar/blob/master/pip/pip-326.md) to align versions. 

Maven:
```xml
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>org.apache.pulsar</groupId>
        <artifactId>pulsar-bom</artifactId>
        <version>3.2.1</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
```

Gradle (groovy syntax):
```groovy
dependencies {
    implementation enforcedPlatform('org.apache.pulsar:pulsar-bom:3.2.1')
}
```

**Reply by lhotari** [reply]
There's now docs for Pulsar BOM: https://pulsar.apache.org/docs/3.2.x/client-libraries-java-setup/#pulsar-bom . Thank you, @onobc
